contracts/Core/BlockManager.sol

```diff
@@ -226,13 +226,13 @@ contract BlockManager is ACL, BlockStorage {
             disputes[epoch][msg.sender].accWeight == voteManager.getTotalStakeRevealed(epoch, assetId),
             "Total stake revealed doesnt match"
         );
-        uint256 median = disputes[epoch][msg.sender].median;
+        uint256 aggregate = disputes[epoch][msg.sender].aggregate;
         uint256 lowerCutoff = disputes[epoch][msg.sender].lowerCutoff;
         uint256 higherCutoff = disputes[epoch][msg.sender].higherCutoff;
         uint256 proposerId = proposedBlocks[epoch][blockId].proposerId;
         //
-        require(median > 0, "Median can't be zero");
-        if (proposedBlocks[epoch][blockId].medians[assetId] != median ||
+        require(aggregate > 0, "Median can't be zero");
```

Comment:
I think we should have "Aggregate can't be zero"

Reply:
Yes. Noted